draft-toorop-dnsop-dns-catalog-zones.md

```diff
@@ -206,9 +207,8 @@ for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers and therefore a value of zero (0) is
 RECOMMENDED.
 
-It is an error for any single owner name within a catalog zone (other
-than the apex of the zone itself) to have more than one RR associated
-with it.
+Each RRSet of catalog zone, with the exception of zone apex, SHOULD consist of just one RR. It's acceptable to generate owner names with the help of
```

Comment:
Perhaps leave the mention of the Hash function, because it is an implementation detail and also the first and only mention that `<m-unique-N>` could be derived in this way.